hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAManagerImpl.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.hadoop.hdds.scm.ha;
 
 import com.google.common.base.Preconditions;
+
```

Comment:
Nit:
if we touch imports here, probably we should move TimeUnit import to java.* imports, otherwise I would just not add a newline here. (Please ignore this if no other changes are done on the PR).

Reply:
I will revert it.